### DISCUSSION TITLE
官网联动计算例子editable和 default input 交替出现Bug

### DISCUSSION THREAD

**Discussion by csc-bo** [alibaba/formily#2738] [Q&A 2.x] [answered]
表单添加行editable和 default input 交替出现
<img width="925" alt="Screen Shot 2022-01-07 at 12 31 58 PM" src="https://user-images.githubusercontent.com/59941818/148492026-4047ef9d-9d87-47e3-8c90-f398445029c8.png">

example：https://codesandbox.io/s/cp0zs

**Comment by janryWang** [comment, marked as answer]
嗯嗯，这个bug已经发现了的，还在想办法修复
